plugins/kubernetes/src/utils/clusterLinks/formatters/aks.test.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { aksFormatter } from './aks';
+
+describe('clusterLinks - AKS formatter', () => {
+  it('should return an url on the workloads when there is a namespace only', () => {
+    expect(() =>
+      aksFormatter({
+        dashboardUrl: new URL('https://k8s.foo.com'),
+        object: {
+          metadata: {
+            name: 'foobar',
+            namespace: 'bar',
+          },
+        },
+        kind: 'Deployment',
```

Comment:
this makes me wonder why the kind isn't part of the object ... which is which and why? :)

Reply:
Yep! I was also surprised that we don't get the 'kind' property in the kubernetes resources but I assumed that it was on purpose since there was another 'kind' property in the KubernetesDrawer component.

So, I took the time to trace the backend code and unfortunately the issue comes from the Kubernetes REST Api which won't provide the explicit 'kind' of resources within a list! It will only return the top level resource's type, contrary to the "kubectl -o json" which does the correct job!
So, I guess that it would deserver a fix in the @kubernetes/client-node module itself!
Otherwise, we could also patch it inside the KubernetesFetcher.ts class...
Your call!